ledger/participant-integration-api/src/main/scala/platform/store/dao/events/ContractsTable.scala

```diff
@@ -68,7 +79,11 @@ private[events] object ContractsTable {
     def execute()(implicit connection: Connection): Unit
   }
 
-  final case class Executables(deleteContracts: Option[BatchSql], insertContracts: Executable)
+  final case class Executables(
+      deleteContracts: Option[BatchSql],
+      insertContracts: Executable,
+      keyNullifies: Option[BatchSql],
```

Comment:
```suggestion
      nullifyPastKeys: Option[BatchSql],
```
consistency with the other fields and the naming in other parts of the code

Reply:
Done